.vscode/settings.json

```diff
@@ -3,14 +3,10 @@
         "*.json5": "jsonc",
         "./kubernetes/**/*.sops.toml": "plaintext"
     },
-    "sops.defaults.ageKeyFile": "age.key",
+    "sops.defaults.ageKeyFile": "age.agekey",
     "yaml.schemas": {
         "Kubernetes": "./kubernetes/*.yaml"
     },
-    "vs-kubernetes": {
-        "vs-kubernetes.kubeconfig": "/workspaces/cluster/kubeconfig",
-        "vs-kubernetes.knownKubeconfigs": ["/workspaces/cluster/kubeconfig"]
-    },
```

Comment:
For v2 of clustertool, we might want to setup multi-cluster kubeconfig and talconfig at the root of the repo.
But for now this is for the best.